.github/workflows/docs.yml

```diff
@@ -39,4 +38,4 @@ jobs:
           git checkout gh-pages
           git pull
           git checkout main
-          mike deploy dev --push --remote https://github.com/online-ml/river.git
+          mike deploy dev --push --remote https://github.com/${{ github.repository }}.git
```

Comment:
The static repo URL always caused builds to fail in my own river fork (i.e., 403 / no authorization), which prevented me from realising when my main builds failed for another reason. So the error went unnoticed when I created the previous PR. :)

Reply:
Nice catch! This probably explains this: https://discord.com/channels/1049726457515102238/1051353581573374103/1061339713228767314